pynxtools/dataconverter/readers/em/reader.py

```diff
@@ -0,0 +1,185 @@
+#
+# Copyright The NOMAD Authors.
+#
+# This file is part of NOMAD. See https://nomad-lab.eu for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Parser for loading generic orientation microscopy data based on ."""
+
+# pylint: disable=no-member,fixme
+
+from typing import Tuple, Any
+
+from pynxtools.dataconverter.readers.base.reader import BaseReader
+# from pynxtools.dataconverter.readers.em.concepts.nxs_concepts import NxEmAppDef
+# from pynxtools.dataconverter.readers.em.subparsers.nxs_mtex import NxEmNxsMTexSubParser
+from pynxtools.dataconverter.readers.em.subparsers.nxs_pyxem import NxEmNxsPyxemSubParser
+# from pynxtools.dataconverter.readers.em.subparsers.nxs_imgs import NxEmImagesSubParser
+# from pynxtools.dataconverter.readers.em.subparsers.nxs_nion import NxEmZippedNionProjectSubParser
+from pynxtools.dataconverter.readers.em.subparsers.rsciio_velox import RsciioVeloxSubParser
+from pynxtools.dataconverter.readers.em.utils.default_plots import NxEmDefaultPlotResolver
+# from pynxtools.dataconverter.readers.em.geometry.convention_mapper import NxEmConventionMapper
+
+# remaining subparsers to be implemented and merged into this one
+# from pynxtools.dataconverter.readers.em_om.utils.generic_eln_io \
+#     import NxEmOmGenericElnSchemaParser
+# from pynxtools.dataconverter.readers.em_om.utils.orix_ebsd_parser \
+#     import NxEmOmOrixEbsdParser
+# from pynxtools.dataconverter.readers.em_om.utils.mtex_ebsd_parser \
+#     import NxEmOmMtexEbsdParser
+# from pynxtools.dataconverter.readers.em_om.utils.zip_ebsd_parser \
+#     import NxEmOmZipEbsdParser
+# from pynxtools.dataconverter.readers.em_om.utils.dream3d_ebsd_parser \
+#     import NxEmOmDreamThreedEbsdParser
+# from pynxtools.dataconverter.readers.em_om.utils.em_nexus_plots \
+#     import em_om_default_plot_generator
```

Comment:
Please remove unused code.

Reply:
intentionally deactivated for debugging but indeed removed or reactivated in pynxtools-em